src/api/index.ts

```diff
@@ -139,7 +139,7 @@ router.get("/", (_: Request, res: Response): void => {
 });
 
 router.get(
-	"/:installationId/:jiraHost/repoSyncState.json",
+	"/:installationId/:jiraHost/syncstate",
```

Comment:
This API is just an Admin API to be used for investigations? Or is it called from somewhere else?

Reply:
only admin API, nothing else uses it.  I wanted to remove confusion where possible.